TS/org.aardappel.TreeSheets.metainfo.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do all these metadata files have to sit in `TS` ? `TS` is also the distribution folder for other platforms, that clearly don't need this file.

Reply:
I put the Linux-only stuff into a separate folder.